### ISSUE TITLE
Possibility to remove files using this action

### ISSUE THREAD

**Issue by didnotwant** [Decathlon/wiki-page-creator-action#11]
I know that this action has "creator" in its name but how about adding a functionality which removes files when they were removed from the repository? Another point of view: it reflects the current state of the markdown files in repository to Wiki.

**Why?**
I’d like to keep all the the history about Wiki changes in pull requests. It’s way more consistent and I only want to introduce every change to Wiki using pull requests. I don’t want to use GitHub’s GUI to perform any change.

Maybe such a possibility exists but I couldn’t find a way to do it?

Maybe it’s out of scope of what this tool was invented for?

What do you think about that?

**Comment by mmornati** [comment]
Indeed it is something we can imagine. Personally I'm not actually using it in this way but linked with the [release notes action](https://github.com/Decathlon/release-notes-generator-action). This means every time I will have a new folder with only a file name to upload to the wiki.

What do you imagine exactly? Link the action to a folder already containing a git repository and just push it?
Can you give me a real example about the usage of this action... and I'll be glad to help you and improve the action :)

**Comment by pgcath** [comment]
Chiming in - Our use case for this action would be similar.  We work on a bunch of different small projects, and use the wiki to keep track of project specific info too verbose for the README.   Rather than editing the wiki through the Github UI, we'd like to be able to just update the wiki's Markdown files in our normal vscode workflow and use wiki-page-creator-action to update the wiki on commit/PR/etc.   

tl;dr - what @didnotwant says above, but some concrete examples from our workflow.

**Comment by Andrew-Chen-Wang** [comment]
@pgcath @didnotwant You can use my GH action here: https://github.com/Andrew-Chen-Wang/github-wiki-action using rsync. Pages are deleted since I'm just linking a directory in a repository (any repository you specify) and moving it to the git source of the wiki. Also, it doesn't have to be Markdown files. Same features and PAT not needed.

Edit: you can still use a PAT. One of the features is rsyncing from a different repository which required a PAT.

**Comment by mmornati** [comment]
Thanks @Andrew-Chen-Wang, indeed I think the rsync it is a quite brilliant solution. I will make an update on our action using it too 👍

**Comment by mmornati** [comment]
Just for your information @Andrew-Chen-Wang, as you are saying that your action is not using the `PAT`. It is mandatory if you are on private repositories as you can't commit without a proper authentication over the https (for example on our enterprise one). That the reason we added it...
Having a PAT will allow you better compatibility...

**Comment by Andrew-Chen-Wang** [comment]
@mmornati Thanks! The action is able to use a PAT. I didn't have a private repository with a PRO account to enable wiki, so I didn't know you needed a PAT, but it works nonetheless for both GitHub token and PAT right now.
